zeeguu_api/api/teacher_dashboard.py

```diff
@@ -31,7 +31,7 @@ def has_permission_for_cohort(cohort_id):
 
 @api.route("/has_permission_for_cohort/<id>", methods=["GET"])
 @with_session
-def has_permission_for_cohorts(id):
+def has_permission_for_cohort_public(id):
```

Comment:
what does public mean here?

Reply:
There is already a function has_permissions_for_cohort that is private and used inside other functions. Whereas the routed has_permissions_for_cohort is mainly used on the dashboard to identify if it should attempt to load something from the API.